packages/moonstone/TooltipDecorator/Tooltip.module.less

```diff
@@ -1,17 +1,58 @@
 // Tooltip.module.less
 //
-@import '../styles/variables.less';
-@import '../styles/mixins.less';
-@import '../styles/skin.less';
+@import "~@enact/moonstone/styles/variables.less";
```

Comment:
Yep, always use relative when accessing something within its own package. We have special handling to cover outside cases, but in general it can break certain things since it's essentially saying to look for an external package dependency (rather than internal).

Reply:
I believe this was a mistake to include "moonstone" in the path. I'll change this to the correct path. Likely a copy/paste issue from my test app.